contracts/escrow/LemonadeEscrowFactoryV1.sol

```diff
@@ -33,6 +57,10 @@ contract LemonadeEscrowFactoryV1 is Ownable, ILemonadeEscrowFactory {
         uint16 hostRefundPercent,
         RefundPolicy[] memory refundPolicies
     ) external {
+        if (_feeAmount > 0 && _feeCollector != address(0)) {
```

Comment:
What is this used for?

Reply:
this feature is requested by KC

Reply:
Make the function payable and verify that the fee is actually paid